src/service/url-replacements-impl.js

```diff
@@ -166,6 +170,20 @@ export class UrlReplacements {
       });
     });
 
+    // Returns assigned variant name for the given experiment.
+    this.set_('VARIANT', experiment => {
+      return this.variants_.then(variants => {
+        user.assert(variants,
```

Comment:
For better or worse, we only report errors for substitutions. This allows e.g. analytics vendors to have generic configuration that works with and without variants.

Return `null` if none is defined.